ansible_runner/__main__.py

```diff
@@ -0,0 +1,153 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+import pkg_resources
+import threading
+import argparse
+import signal
+import errno
+import json
+import stat
+import sys
+import os
+import shlex
+
+from uuid import uuid4
+
+from ansible_runner import run, run_async
+
+VERSION = pkg_resources.require("ansible_runner")[0].version
+DEFAULT_ROLES_PATH = os.getenv('ANSIBLE_ROLES_PATH', '/etc/ansible/roles')
```

Comment:
This should default to our project base directory (aka private_data_dir) and can also include `/etc/ansible/roles`

Something like `{project}/roles` ?

Reply:
In much the same way as we allow `RUNNER_PLAYBOOK` we should also allow `RUNNER_ROLE` also.

Reply:
Those would be mutually exclusive I would think.

Adding a demo role would be hot also ;)